src/Api/Mastodon/Account.php

```diff
@@ -80,6 +82,12 @@ public static function createFromContact(array $contact) {
 		$account->header_static = '';
 		// No custom emojis per account in Friendica
 		$account->emojis = [];
+		$account->bot = ($contact['contact-type'] == Contact::TYPE_NEWS);
+
+		$apcontact = APContact::getByURL($contact['url'], false);
+		if (!empty($apcontact)) {
+			$account->locked = (bool)$apcontact['manually-approve'];
```

Comment:
These three line can be simplified to just 

```
$account->locked = !empty($apcontact['manually-approve']);
```
This way it can be hoisted on line 70 and the `$apcontact = APContact::getByURL($contact['url'], false);` line 65. And the new line I forgot can be added before the method opening brace.